internal/configs/version2/nginx-plus.virtualserver.tmpl

```diff
@@ -423,6 +423,11 @@ server {
         {{ $proxyOrGRPC }}_set_header username $jwt_claim_sub;
         {{ end }}
 
+        {{ if $s.OIDC.AccessTokenEnable }}
```

Comment:
This could cause a nil pointer exception, we should check if OIDC is set first like we do above
```
{{ with $oidc := $s.OIDC }}
```

Reply:
Thanks for your feedback. I have addressed it. Let me know if I miss anything.